index.js

```diff
@@ -900,7 +904,9 @@ function modifyFirstrun() {
                 // sidebar to be shown.
                 startNotificationTimer(1);
                 // And store our onboarding initialization time so that we can set a new timer for the first notification if the user exits and restarts the session
-                simpleStorage.lastSidebarLaunchTime = Date.now();
+                simpleStorage.lastSidebarCTACompleteTime = Date.now();
+                // note that we haven't sent our notification for this sidebar yet
+                simpleStorage.shownNotification = false;
```

Comment:
starting the onboarding, so we need to mark that we haven't yet been shown our first notification